packages/js/data/src/utils.ts

```diff
@@ -21,6 +21,25 @@ export function getResourceName(
 	return `${ prefix }:${ identifierString }`;
 }
 
+/**
+ * Generate a resource name for order totals count.
+ *
+ * It omits query parameters from the identifier that don't affect
+ * totals values like pagination and response field filtering.
+ *
+ * @param {string} prefix Resource name prefix.
+ * @param {Object} query  Query for order totals count.
+ * @return {string} Resource name for order totals.
+ */
+export function getTotalCountResourceName(
+	prefix: string,
+	query: Record< string, unknown >
```

Comment:
Why type `query` as `Record< string, unknown >` here instead of `Partial< ItemQuery >`?

Reply:
`ItemQuery` is unique to the `crud` datastore, given this util is in the main data package. `Record< string, unknown >` is more generic. I followed the same format as `getResourceName` above.